soe/src/sites/default/settings.php

```diff
@@ -1,15 +1,23 @@
 <?php
 
+//Pull Acquia Global environments.
+if (file_exists('/var/www/site-php')) {
+  require '/var/www/site-php/swssoe/swssoe-settings.inc';
```

Comment:
Interdasting 🤔 . So that file sets the environment variables, but it must get called at runtime, because the environment variables are already set. I don't think this is necessary - just belt and suspenders?

Reply:
IDK. According to the docs, you need to add it to the settings file before calling the env? They even labeled it 'important.'

https://docs.acquia.com/cloud-platform/develop/env-variable/